frontend/src/pages/ImportCkan/SelectTagsScreen.jsx

```diff
@@ -0,0 +1,215 @@
+    padding: 0.5em;
+    margin: 1em;
+    height: 2em;
+    display: table-cell;
+    width: 20em;
+`;
+
+const ButtonPart = styled.div`
+    float: right;
+`;
+
+const LabelPart = styled.div`
+    position: absolute;
+    float: left;
+    line-height: 2em;
+`;
+
+const ImportButton = styled.div`
+    margin: 2px;
+    border-radius: 4px;
+    border: 1px solid #000000;
+    background-color: #aaaaaa;
+    padding: 0.2em;
+`;
+
+const NavigationContainer = styled.div`
+    display: flex;
+
+`;
+
+const TagEntry = ({ tag, toggleEntry, index }) => {
+  let existsButton = (<b>Already exists</b>);
+  if (!tag.alreadyExists) {
+    if (tag.import) {
+      existsButton = (<ImportButton style={{ backgroundColor: '#66aa66' }}>Import</ImportButton>);
+    } else {
+      existsButton = (<ImportButton style={{ backgroundColor: '#aa6666' }}>Don&apos;t import</ImportButton>);
+    }
+  }
+  return (
+    <TagContainer onClick={() => { toggleEntry(index); }}>
+      <LabelPart>
+        {tag.name}
+      </LabelPart>
+      <ButtonPart>
+        {
+          existsButton
+        }
+      </ButtonPart>
+    </TagContainer>
+  );
+};
+TagEntry.propTypes = {
+  tag: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+    alreadyExists: PropTypes.bool.isRequired,
+    import: PropTypes.bool.isRequired,
+  }).isRequired,
+  toggleEntry: PropTypes.func.isRequired,
+  index: PropTypes.number.isRequired,
+};
+
+
+export const SelectTagsScreen = ({ tags, nextState, setTags }) => {
+  const [loading, setLoading] = useState(false);
+  const [offset, setOffset] = useState(0);
+
+  const PAGE_SIZE = 20;
+
+  const fail = () => {
+    setLoading(false);
+  };
+
+  const nextPage = () => {
+    if (offset + PAGE_SIZE < tags.length) {
+      setOffset(offset + PAGE_SIZE);
+    }
+  };
+
+  const prevPage = () => {
+    setOffset(offset < PAGE_SIZE ? 0 : offset - PAGE_SIZE);
+  };
+
+  const toggleEntry = (index) => {
+    setTags(tags.map((element, i) => {
+      if (index === i) {
+        const newElement = element;
+        newElement.import = !element.import;
+        return newElement;
+      }
+      return element;
+    }));
+  };
+
+  const massUpdate = (newValue) => {
+    setTags(tags.map((tag) => {
+      const newTag = tag;
+      newTag.import = newValue;
+      return newTag;
+    }));
+  };
+
+  const userSelector = useSelector((state) => state.user);
+  const { token } = userSelector.user;
+
+  const doTagUpdate = async () => {
+    setLoading(true);
+    try {
+      /* eslint-disable no-await-in-loop */
+      for (const tag of tags) {
+        if (tag.import === true && !tag.alreadyExists) {
+          await (await fetch('/api/Tag', {
+            method: 'PUT',
+            headers: {
+              'Content-Type': 'application/json',
+              Authorization: `bearer ${token}`,
+            },
+            body: JSON.stringify({ name: tag.name }),
+          })).json();
+        }
+      }
+      /* eslint-enable no-await-in-loop */
```

Comment:
I will not stop suggesting these btw.
```suggestion
      for (const tag of tags) {
        if (tag.import === true && !tag.alreadyExists) {
          // eslint-disable-next-line no-await-in-loop
          await (await fetch('/api/Tag', {
            method: 'PUT',
            headers: {
              'Content-Type': 'application/json',
              Authorization: `bearer ${token}`,
            },
            body: JSON.stringify({ name: tag.name }),
          })).json();
        }
      }
```

Reply:
The linter does not like it

Reply:
I don't get any issues with this, can you show me the exact error?

Reply:
Note: The `eslint-disable-next-line` comment is not exactly in the same position as you put `eslint-disable`. It was moved down to the line before the `await`

Reply:
I committed an example. See LoadFromCkanScreen.jsx L83